src/lib/y2storage/planned_volume.rb

```diff
@@ -187,9 +187,35 @@ def create_filesystem(partition)
     # if this is a btrfs root filesystem.
     #
     # @param filesystem [::Storage::Filesystem]
-    def create_subvolumes(filesystem)
+    # @param other_mount_points [Array<String>]
+    #
+    # @return nil
+    #
+    def create_subvolumes(filesystem, other_mount_points)
       return unless filesystem.type == ::Storage::FsType_BTRFS
       return unless subvolumes?
+      parent_subvol = get_parent_subvol(filesystem)
+      prefix = filesystem.mountpoints.first
+      prefix += "/" unless prefix == "/"
```

Comment:
In this case I do want a trailing slash, but not a double one if the main volume's mount point is "/" (which is the common case for our proposed subvolumes).